packages/gatsby-theme-iterative/config/prismjs/cml-commands.js

```diff
@@ -1,9 +1,9 @@
 module.exports = [
   'ci',
-  'pr',
-  'publish',
   'runner',
-  'send-comment',
-  'send-github-check',
-  'tensorboard-dev'
+  'pr',
+  'comment',
+  'check',
+  'workflow',
+  'tensorboard'
```

Comment:
are subcommands meant to be included? If so...

```suggestion
  'runner launch',
  'pr',
  'pr create',
  'comment',
  'comment create',
  'comment update',
  'check',
  'check create',
  'workflow',
  'workflow rerun',
  'tensorboard'
  'tensorboard connect'
```

Reply:
Yes, we would want subcommands as well to highlight them properly. 
Can we do it organize the subcommands similarly to dvc.org (eg: [`dvc data status`](https://dvc.org/doc/command-reference/data/status) ) to ensure consistency and also it will provide better navigation on the sidebar?

Reply:
Unlike DVC, atm it doesn't make sense for CML subcommands to have separate pages.

i.e. https://dvc.org/doc/command-reference/exp/init and https://dvc.org/doc/command-reference/exp/run are big enough to be on separate pages, but https://cml.dev/doc/ref/comment#create and https://cml.dev/doc/ref/comment#update should be on the same page - trying to separate them would harm reader experience.

Reply:
If we ensure consistency, it will be picked up by the GHA automation.

> `create` and `update` should be on the same page - trying to separate them would harm reader experience.

This seems to be a compromise we can work on(linking each other with a little more description) if we want to use the automation to generate the commands list upon addition. 

In a long run, we want to have these commands listed on the consumer project themself instead of on the theme project (https://github.com/iterative/gatsby-theme-iterative/issues/134).

Reply:
what's the "GHA automation" - can you provide a link to some code/logic?

Reply:
This PR was generated by https://github.com/iterative/gatsby-theme-iterative/blob/main/.github/workflows/update-commands.yml.
[The script](https://github.com/iterative/gatsby-theme-iterative/blob/main/packages/gatsby-theme-iterative/config/prismjs/get-commands.js) basically gets all the commands from the sidebar.json from the respective tool's repository and updates the command list.